package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "eslint \"**/*.{js,jsx,ts,tsx}\"",
     "lint:fix": "eslint --fix \"**/*.{js,jsx,ts,tsx}\"",
     "mirror-box": "ts-node etc/mirror-box.ts",
-    "netlify-export": "yarn mirror-box && yarn build && next export",
+    "netlify-export": "yarn fetch-wbw && yarn build && next export",
```

Comment:
@adityapurwa, please pull the latest `main` branch to your [wbw-box](https://wbw-box.lucentshard.com/) instance once this PR is merged, thanks! 🙏